randovania/games/samus_returns/exporter/patch_data_factory.py

```diff
@@ -58,18 +115,78 @@ def _start_point_ref_for(self, node: Node) -> dict:
         else:
             return {}
 
+    def _key_error_for_node(self, node: Node, err: KeyError):
```

Comment:
Add a return type hint  here

Reply:
I'm not sure what you mean

Reply:
`def _key_error_for_node(self, node: Node, err: KeyError) -> KeyError:`